source/unified-test-format/tests/valid-pass/poc-queryable-encryption.yml

```
@@ -0,0 +1,85 @@
+description: poc-queryable-encryption
+
+schemaVersion: "1.23"
+
+runOnRequirements:
+  - minServerVersion: "7.0"
+    csfle: true
+
+createEntities:
+  - client:
+      id: &client0 client0
+      observeEvents:
+        - commandStartedEvent
+        - commandSucceededEvent
+      autoEncryptOpts:
+        keyVaultNamespace: keyvault.datakeys
+        kmsProviders:
+          local:
+            key: Mng0NCt4ZHVUYUJCa1kxNkVyNUR1QURhZ2h2UzR2d2RrZzh0cFBwM3R6NmdWMDFBMUN3YkQ5aXRRMkhGRGdQV09wOGVNYUMxT2k3NjZKelhaQmRCZGJkTXVyZG9uSjFk
+  - database:
+      id: &encryptedDB encryptedDB
+      client: *client0
+      databaseName: &encryptedDBName poc-queryable-encryption
+  - collection:
+      id: &encryptedColl encryptedColl
+      database: *encryptedDB
+      collectionName: &encryptedCollName encrypted
+  - database:
+      id: &keyvault keyvaultDB
+      client: *client0
+      databaseName: &keyvaultName keyvault
+  - collection:
+      id: &datakeys datakeysColl
+      database: *keyvault
+      collectionName: &datakeysName datakeys
+
+
+initialData:
+  - databaseName: *keyvaultName
+    collectionName: *datakeysName
```

```suggestion
initialData:
  - databaseName: keyvault
    collectionName: datakeys
```

Suggested simplification. Creating a collection entity for the key vault is not needed since the data is inserted in `initialData`.